index-api/src/Ergvein/Index/API/Models.hs

```diff
@@ -0,0 +1,15 @@
+module Ergvein.Index.API.Models where
+
+import Data.Word
+import Data.Text
+import Ergvein.Aeson
+
+type LockingScriptHash = Text
+
+data Currency = BTC | ERGO
```

Comment:
Lets factor out to new module Ergvein.Types.Currency, will be used in other places too.